xml/signals/SPTCO-1930/appearance-DW-2-head.xml

```diff
@@ -114,22 +114,27 @@
     <aspectMapping>
       <advancedAspect>Diverging Clear</advancedAspect>
       <ourAspect>Clear</ourAspect>
+      <ourAspect>Diverging Clear</ourAspect>
```

Comment:
Perhaps I'm not understanding the case you're asking about. Consider this case:
![skitch](https://cloud.githubusercontent.com/assets/2774117/26522653/e14dc300-42b9-11e7-8444-0a7b92fa4f3b.png)
(ignore the aspects being shown, it's from another example) The blue arrow points to the signal I'm asking about.  The yellow arrow points to a signal which is showing "Diverging Clear" because the routing is set through a high-speed turnout.

What should the blue-arrow signal show?